src/components/shared/forms/ProjectDetailsForm.tsx

```diff
@@ -0,0 +1,77 @@
+import { Button, Form, FormInstance } from 'antd'
+import { t, Trans } from '@lingui/macro'
+
+import { FormItems } from 'components/shared/formItems'
+import { normalizeHandle } from 'utils/formatHandle'
+import { cidFromUrl, unpinIpfsFileByCid } from 'utils/ipfs'
+
+export type ProjectDetailsFormFields = {
+  name: string
+  description: string
+  infoUri: string
+  handle: string
+  logoUri: string
+  twitter: string
+  discord: string
+  payButton: string
+  payDisclosure: string
+}
+
+export default function ProjectDetailsForm({
```

Comment:
This component is mostly the same is previously. I'll point out the specific changes below.